docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1,195 @@
+* software-properties-common
+* php7.2-mysql
+* php7.2-xml
+* php7.2-json
+* php7.2-mbstring
+* php7.2-gd
+* libapache2-mod-php7.2
+
+### Database set up
+
+MySQL (or MariaDB) must be installed and a root or admin-level MySQL user must
+be created before continuing. (This is not the same as a unix root credential.)
+
+## Getting the source code
+
+### LORIS for Development
+If installing LORIS for development purposes, clone the source code from our
+GitHub repository to the web root. Then change to this directory.
+
+Example command assuming LORIS will be served from `/var/www/`:
+
+```bash
+git clone git@github.com:aces/Loris.git /var/www/loris
+cd /var/www/loris/
+```
+
+### LORIS for Production
+
+Visit the releases page and download the zipped file of the latest LORIS
+release. The below command will download the latest LORIS release and save 
+it to a file named `loris-src.tar.gz`.
+
+```bash
+curl -s https://api.github.com/repos/aces/loris/releases/latest \
+| grep 'tarball' \
+| cut -d : -f2,3 \
+| tr -d \" \
+| tr -d , \
+| wget -i - -O loris-src.tar.gz
+```
+
+When this is complete, expand the compressed file and move its contents to the
+directory from which LORIS will be served. 
+
+```bash
+tar -zxf loris-src.tar.gz
+```
+
+This will create a folder called something like `aces-Loris-9e30cf0`. (The
+part after `aces-Loris` is not important).
+
+Rename this file to `loris`, e.g.
+
+```bash
+mv aces-Loris-9e30cf0 loris
+```
+
+Then move this folder to the web root (e.g. `/var/www/`) and go to this 
+directory.
+
+```
+mv ./loris /var/www/loris/
+cd /var/www/loris/
+```
+
+## Running the install script
+
+The next step in setting up LORIS is running the script `install.sh` in the 
+`tools/` directory. The script must be run from this directory.
+
+This will begin an interactive setup process that will configure files and
+permissions required to get LORIS up and running.
+
+The install script will prompt for a `root` password in order to configure
+certain file permissions as well as Apache. However, the script should not
+be run as `root` and will exit automatically if it detects that it has been
+executed with superuser permissions.
+
+```bash
+cd tools/
+./install.sh
+```
+
+## Compiling source code and installing dependencies
+Once this step is complete, run the `make` command to install additional
+dependencies and compile JavaScript needed to properly render the LORIS
+front-end.
+
+`make` must be run from the LORIS root, not `tools/`
+
+### Installing LORIS for production
+
+```bash
+cd `/var/www/loris/`; make
+```
+
+### Installing LORIS for development
+
+```bash
+# This is necessary for 'phan', a static analysis tool
+sudo pecl install ast-1.0.3
+cd `/var/www/loris/`; make dev
+```
+
+The `dev` target for `make` will install additional libraries that are needed 
+for development process but not for production installs.
+
+## Configuring the database
+
+Open your browser and go to: `http://%IPADDRESS%/installdb.php` where 
```

Comment:
Why IP address ? It could be domain name

Reply:
True, I'll change this